File: C/斐波那契数列.c
#include<stdio.h>
#include<stdlib.h>
long long fib(int n)
{
	if(n<1||n>45) return -1;
	else if(n==1||n==2) return 1;
	else return fib(n-1)+fib(n-2);
} 
int main(int argc,char** argv)
{
	FILE *fp;
	if((fp=fopen("result.txt","w"))==NULL)
		return -1;
	int x;
	long long an;
	x=atoi(argv[1]);
	an=fib(x);
	if(an==-1) fprintf(fp,"I");
	else fprintf(fp,"%ld",an);
	fclose(fp);	
	return 0;
}

File: Python/斐波那契数列.py
import sys
def fib(x):
	if x<1 or x>45 :
		return -1
	elif x==1 or x==2 :
		return 1
	else :
		return fib(x-1)+fib(x-2)
x=int(sys.argv[1])
y=fib(x)
with open('result.txt','w') as fp:
	if y==-1 :
		fp.write('I')
	else :
		fp.write(str(y))